buildbots.rst

```diff
@@ -209,7 +209,7 @@ Custom builders
 When working on a platform-specific issue, you may want to test your changes on
 the buildbot fleet rather than just on Travis and AppVeyor.  To do so, you can
 make use of the `custom builders
-<http://buildbot.python.org/all/waterfall?category=custom.stable&category=custom.unstable>`_.
+<http://buildbot.python.org/all/#/waterfall?category=custom.stable&category=custom.unstable>`_.
```

Comment:
This whole link doesn't really work anymore.  We could try `http://buildbot.python.org/all/#/builders?tags=custom.unstable&tags=custom.stable` instead.